cql/ManageSpecialPopulation.cql

```diff
@@ -236,16 +246,18 @@ define ShouldSwitchToRiskAt25Text:
 define RecommendationForPatientsYoungerThan25:
   case
     // Heading 3: Management of Histology of Less than CIN2 Preceded by Cytology ASC-H and HSIL in Patients Younger than 25 Year
-    when ( // Heading 3.3
+    // ===========    
+    when ( // K.1 Heading 3 case 1
```

Comment:
Should we designate that these are K.1, Heading 3, part 3, and label as K.1.3.3, to align with how you have labelled part 1 and part 2?

Reply:
This makes things a lot clearer, thanks!